commit/core.pyx

```diff
@@ -491,152 +491,149 @@ cdef class Evaluation :
         LOG( '   [ %.1f seconds ]' % ( time.time() - tic ) )
 
 
-    def set_threads( self, n=None ) :
+    def set_threads( self, nthreads = None, select_gpu = 0 ) :
```

Comment:
Renaming variable `n` to `nthreads` may create incompatibilities with older scripts.
@daducci do you know if this parameter has been used explicitly?  do you think it is OK to change it?